rules/no-large-snapshots.js

```diff
@@ -11,7 +11,11 @@ module.exports = {
   create(context) {
     if (context.getFilename().endsWith('.snap')) {
       const lineLimit =
-        (context.options[0] && context.options[0].maxSize) || 50;
+        context.options[0] &&
+        typeof context.options[0].maxSize === 'number' &&
+        isFinite(context.options[0].maxSize)
```

Comment:
Could we do `Number.isFinite`? Then you can drop the `typeof` check 🙂 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite#Description

Reply:
Sure thing!